CMakeLists.txt

```diff
@@ -3,14 +3,14 @@ cmake_minimum_required(VERSION 2.8.9)
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 
-set(PROJECT_VERSION "2.1.0")
+set(PROJECT_VERSION "2.0.95")
```

Comment:
URGH, why lower the version numer?

Reply:
ANTI-URGH: I saw that, but it wasn't me! I always fetch+rebase upstream before making changes. It either came from master or my cmake did it...

Reply:
@tari01: Hmmm... it must not be in this PR, then. I will amend the commit before merging.